website/docs/whats-new/release-notes/partials/figma-library-components.md

```diff
@@ -12,6 +12,27 @@
 </p>
 
 
+### August 2nd, 2024
+
+`ApplicationState` - multiple enhancements to the `ApplicationState` including:
+
+- Added support for a media slot, which when enabled, will be added above the title, allowing yielded illustrations to occupy that space.
+- Added an `alignment` property which can be set at the root level to `left` or `center`. This adjusts the component’s text, footer actions, and media alignment.
+- The footer now supports up to three actions at once, modeled after the Button Set. Previously the actions were justified (one all the way left and one all the way right). Supported actions now include Dropdown, Button, and Standalone Link. Previously Standalone Link was the only supported option.
+
+Notable visual changes in the `ApplicationState`:
+
+- The divider has been removed to modernize the component and align with our visual standards.
+- The title text will be reduced from Display 400 to Display 300 to better fit within the page hierarchy.
```

Comment:
```suggestion
- The title text has been reduced from Display 400 to Display 300 to better fit within the page hierarchy.
```

Reply:
suggestion: only operate documentation changes in `packages/components/CHANGELOG-FIGMA-COMPONENTS.md`, this file is automatically generated, so the changes addressed with this comment will be overwritten